zerver/actions/create_realm.py

```diff
@@ -295,11 +296,17 @@ def do_create_realm(
     )
     realm.signup_announcements_stream = signup_announcements_stream
 
+    # New realm is initialized with the latest zulip updates level
+    # as it shouldn't receive a bunch of old updates.
+    # Should we send a DM to admins, if they want to change the stream for updates?
```

Comment:
I think we don't need DMs here, but we could consider sending including in our onboarding notices a message to a topic in the default value of this stream for new organizations; @alya FYI as something to consider in your onboarding project but we can probably keep it out of this PR in any case.

Reply:
Oh looks like you already have one in `onboarding.py`, so yeah I think you can delete this comment.

Reply:
Yeah, I just forgot to remove that comment. Sorry for the confusion.